getting-started/getting-started.md

```diff
@@ -62,6 +84,8 @@ repositories {
 rewrite {
     // Will configure in subsequent steps
 }
+
+// ...
 ```
 {% endcode %}
 
```

Comment:
I didn't follow the hint below when I was reading through this guide but I left it in as I presumed it was important. If one of y'all could take a look and double-check (and perhaps provide me with some additional context) that would be swell!

```When applied to a multi-project build, behavior differs depending on whether the plugin is applied to the root project or to a sub-project. Applied to the root project, the plugin will parse and refactor all sources from all projects. Applied to any project other than the root project, the plugin will parse and refactor only sources from that project.```

Reply:
I think this is really the expected behavior (of any gradle build task): 

if you run `./gradlew rewriteDryRun`, I think most developers would expect that to work on the entire project, whereas, `./gradlew :subProject:rewriteDryRun` would just work on the subproject. 

Not sure if there is something specific we are trying to call out here. I might ask @sambsnyd

Reply:
That note could probably be mentioned only in the documentation on the gradle plugin itself, rather than here. Someone following our prescriptive guidance would not have to make any decision around this

Reply:
Alrighty -- I removed it to reduce confusion. Thanks!